tests/xharness/AppRunner.cs

```diff
@@ -887,7 +887,7 @@ public async Task<int> RunAsync ()
 					FailureMessage = $"Launch failure";
 					if (Harness.InCI)
 						XmlResultParser.GenerateFailure (Logs, "launch", appName, Variation, $"AppLaunch on {device_name}", $"{FailureMessage} on {device_name}", main_log.FullPath, XmlResultParser.Jargon.NUnitV3);
-				} else if ((!File.Exists (listener_log.FullPath) || string.IsNullOrEmpty (crash_reason)) && Harness.InCI) {
+				} else if (crashed && (!File.Exists (listener_log.FullPath) || string.IsNullOrEmpty (crash_reason)) && Harness.InCI) {
```

Comment:
That's different from
```
				} else if ((crashed && !File.Exists (listener_log.FullPath) || (!crashed && string.IsNullOrEmpty (crash_reason)) && Harness.InCI)) {
```
in your other PR...

also the screenshot shows the error that's not related to the false crash report ;-)

Reply:
@spouliot  I got it wrong in the other PR, that's why I triple tested :/ 

Yeah screenshot is just to show it does not crash.